core/src/main/scala/doobie/free/blob.scala

```diff
@@ -156,14 +156,7 @@ object blob {
    * a `java.sql.Blob` and produces a value of type `A`. 
    * @group Algebra 
    */
-  type BlobIO[A] = F.FreeC[BlobOp, A]
-
-  /**
-   * Monad instance for [[BlobIO]] (can't be inferred).
-   * @group Typeclass Instances 
-   */
-  implicit val MonadBlobIO: Monad[BlobIO] = 
-    F.freeMonad[({type λ[α] = Coyoneda[BlobOp, α]})#λ]
+  type BlobIO[A] = F[BlobOp, A]
```

Comment:
I don't really like aliasing `Free[_, _]` to `F[_, _]` here and there, but I'll keep it for now otherwise it conflicts with `blob.Free` op.